components/ingest-service/pipeline/publisher/config_mgmt_publisher.go

```diff
@@ -2,11 +2,14 @@ package publisher
 
 import (
 	"context"
+	"encoding/json"
 	"time"
 
 	"github.com/chef/automate/api/interservice/cfgmgmt/request"
 	cfgmgmt "github.com/chef/automate/api/interservice/cfgmgmt/service"
 	"github.com/chef/automate/components/ingest-service/pipeline/message"
+
+	"github.com/chef/automate/components/ingest-service/backend"
```

Comment:
This parses the cookbook locks out of the policy update action. This information is sent to the config-mgmt-service and stored in PostgreSQL.